docs/data/material/components/autocomplete/GloballyCustomizedOptions.tsx

```diff
@@ -0,0 +1,652 @@
+import * as React from 'react';
+import {
+  Autocomplete,
+  Box,
+  Button,
+  createTheme,
+  Stack,
+  TextField,
+  ThemeProvider,
+  useTheme,
+  Theme,
+} from '@mui/material';
+
+// Theme.tsx
+const customTheme = (outerTheme: Theme) =>
+  createTheme({
+    palette: {
+      mode: outerTheme.palette.mode,
+    },
+    components: {
+      MuiAutocomplete: {
+        defaultProps: {
+          renderOption: (props, option, state, ownerState) => (
+            <Box sx={{ padding: '0 !important' }} component="li" {...props}>
+              <Button
+                sx={{
+                  fontSize: '12px',
+                  width: '100%',
+                  height: '100%',
+                  padding: '20px 24px',
+                  justifyContent: 'flex-start',
+                  textAlign: 'left',
+                }}
+              >
+                {ownerState.getOptionLabel?.(option)}
+              </Button>
+            </Box>
+          ),
+        },
+      },
+    },
+  });
+
+export default function GlobalCustomizedOptions() {
+  // useTheme is used to determine wether dark or light mode is currently active.
+  const outerTheme = useTheme();
+
+  return (
+    <ThemeProvider theme={customTheme(outerTheme)}>
+      <Stack spacing={5} sx={{ width: 300 }}>
+        <MovieSelect />
+        <CountrySelect />
+      </Stack>
+    </ThemeProvider>
+  );
+}
+
+function MovieSelect() {
+  return (
+    <Autocomplete
+      options={top100Films}
+      getOptionLabel={(option: FilmOptionType) => `${option.title} (${option.year})`}
+      id="movie-customized-option-demo"
+      disableCloseOnSelect
+      renderInput={(params) => (
+        <TextField {...params} label="Choose a movie" variant="standard" />
+      )}
+    />
+  );
+}
+
+function CountrySelect() {
```

Comment:
I believe it's not possible here since to do that can only be done through `renderOption`. To change the `renderOption` would mean also to change `renderOption` for `MovieSelect`, which we don't want. `getOptionLabel` is used to determine the string value of the option.